jenkins.go

```diff
@@ -503,7 +512,7 @@ func CreateJenkins(base string, auth ...interface{}) *Jenkins {
 		base = base[:len(base)-1]
 	}
 	j.Server = base
-	j.Requester = &Requester{Base: base, SslVerify: false}
+	j.Requester = &Requester{Base: base, SslVerify: true}
```

Comment:
Sorry for the confusion. I mean that j.Requester.CACert is not initialized, and find no place to set its value, so its value is always `nil`.